schema/deploy/tables/revision_status.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy cif:tables/revision_status to pg
+
+begin;
+
+alter table cif.amendment_status rename to revision_status;
+alter table cif.revision_status add column is_amendment_specific boolean;
+
+update cif.revision_status set is_amendment_specific = false where name='Draft' or name='Applied';
+update cif.revision_status set is_amendment_specific = true where name not in('Draft', 'Applied');
+
+alter table cif.project_revision rename amendment_status to revision_status;
```

Comment:
Was there a reason why the changes to project_revision are not in the `tables/project_revision_003_set_amendment_status.sql` file instead - or in a subsequent migration if we need to do the work in a particular order?

Reply:
No good reason! I just got confused by which migration "the migration" in the AC was referring to

Reply:
This line needs to be moved to a migration for the `project_revision` table